models/endpoints.models.js

```diff
@@ -0,0 +1,13 @@
+const fs = require("fs/promises");
+
+exports.readEndpoints = () => {
+  return fs
+    .readFile(`./endpoints.json`, "utf8")
+    .then((data) => {
+      const parsedData = JSON.parse(data);
+      return { endpoints: parsedData };
+    })
```

Comment:
This is fine, but we technically don't need a model here. We can just require the json file into the controller and send it from there.